pyaedt/modules/Boundary.py

```diff
@@ -330,6 +331,19 @@ def delete(self):
         return True
 
 
+def disable_auto_update(func):
+    def wrapper(self, *args, **kwargs):
+        auto_update = self.auto_update
+        self.auto_update = False
+        out = func(self, *args, **kwargs)
+        self.update()
+        self.auto_update = auto_update
+        if out is not None:
+            return out
```

Comment:
Is there a reason for this if statement ? Why wouldn't you return `None` if that's what the function returns ? Btw, that's the basic return value of any function in python.

Reply:
You are absolutely right